pkgs/shells/fish/plugins/editor-updater.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildFishPlugin, fetchFromGitHub }:
+
+buildFishPlugin rec {
+  pname = "editor-updater";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "dudeofawesome";
+    repo = "fish-plugin-editor-updater";
+    rev = "v${version}";
```

Prefixing with `refs/tags` avoids collision with branches
```suggestion
    rev = "refs/tags/v${version}";
```